x/circuit/README.md

```diff
@@ -13,13 +13,13 @@ The transactions are checked and can be rejected at two points:
 * In `CircuitBreakerDecorator` [ante handler](https://docs.cosmos.network/main/learn/advanced/baseapp#antehandler):
 
 ```go reference
-https://github.com/cosmos/cosmos-sdk/blob/x/circuit/v0.1.0/x/circuit/ante/circuit.go#L27-L41
+https://github.com/cosmos/cosmos-sdk/blob/release/v0.52.x/x/circuit/ante/circuit.go#L26-L44
```

Comment:
We probably need to tag a new x/circuit version instead

Reply:
True! 0.2.0 tags of modules are coming after RC

Reply:
Ok, so... should we wait for the new version of the circuit to update this readme? @facundomedica @julienrbrt

Reply:
No, that's an after release todo, it's fine now and more accurate than keeping v0.1